lib/bank.rb

```diff
@@ -0,0 +1,41 @@
+puts 'Your current balance is?'
+balance = gets.chomp.to_i
+
+done='no'
+until done == 'yes' do
+
+
+
+    puts 'What would you like to do?(deposit, withdraw, check_balance)'
+    action = gets.chomp
+
+    if action == 'deposit'
+      puts 'How much would you like to deposit?'
+      deposit = gets.to_i
+
+    elsif action == 'withdraw'
+      puts 'How much would you like to withdraw?'
+      deposit = gets.to_i * -1
```

Comment:
should've used a better naming for the variable here, because we're not depositing anymore